src/main/java/Pages/HomePage.java

```diff
@@ -0,0 +1,14 @@
+package Pages;
+
+import com.codeborne.selenide.SelenideElement;
+import org.openqa.selenium.By;
+import static com.codeborne.selenide.Selenide.*;
+
+public class HomePage {
+
+    String homeLocator = "user";
```

Comment:
no need to name it 'locator', just short understandable name, here I think just 'home' is fine